packages/babel-plugin-strip-runtime/index.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * This exists for local sessions so source files don't need to be built.
+ * This is an unfortunate hack if you're reading this and have a spare
+ * eng health week please find a better way to not have to do this.
+ */
+module.exports = require('./src');
```

Comment:
One of the worst parts of this fix. These are scattered around the repo, used for picking up entrypoints that aren't yet built. Each case of this has a pairing jsdoc with the explanation.

Would love to not need to do this if anyone has ideas!

Reply:
I think you would need something like https://www.npmjs.com/package/babel-plugin-module-resolver to fix it

Reply:
Yeah, wouldn't want to use it yet since TSC is used to build dist. Fodder for consolidating on a single build tool though.